### PR TITLE
Refactor and update

### PR DESCRIPTION
**Pull request by harveyvanveltom**
refactors some unclean code and updated the android test suite.

### CODE REVIEW

File: EscapeApp/app/src/androidTest/java/com/context/kroket/escapeapp/mainscreens/GameOverTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+@RunWith(AndroidJUnit4.class)
+@LargeTest
+public class GameOverTest {
+
+    @Rule
+    public IntentsTestRule<MainActivity> myActivityRule
+            = new IntentsTestRule<MainActivity>(MainActivity.class);
+
+    //Test if the game over xml is displayed correctly
```
Comment (Mayke93) [inline EscapeApp/app/src/androidTest/java/com/context/kroket/escapeapp/mainscreens/GameOverTest.java]: This should be with javadoc, also at other places

**Comment by Mayke93** [comment]
I am missing some javadoc. But I see you have removed the testing boolean etc., this is good. The travis passes, so I think this can be merged.

**Comment by irenevanderblij** [comment]
Great job on applying the feedback. The test succeed, so that is good.

Is everything that can be tested now tested?

**Comment by harveyvanveltom** [comment]
Updated javadoc Mayke

no Irene not every class is tested yet. but the majority of classes are. adding the last tests won't be too difficult so i'll have that done by next week.

**Pull request merged**